tests/shutdown/shutdown.pm

```diff
@@ -15,7 +15,7 @@ use utils;
 sub run {
     my $self = shift;
     select_console('root-console');
-    systemctl 'list-timers --all';
+    systemctl('list-timers --all', timeout => 500);
```

Comment:
Same as above. I would set some base e.g  60 and then let TIMEOUT_SCALE to multiply the original value if needed

Reply:
As the same, due to current situation - low performance, we plan to finish all test module for ppc64le image firstly. Draft this PR as purpose. Next, will do more validation run and continue to make more code enhancement about ppc64le test run for SL Micro6.1

Reply:
Try to set up TIMEOUT_SCALE=30 to solve the timeout for the whole test suite